sim/src/main/makefrag/firesim/Makefrag

```diff
@@ -89,18 +89,19 @@ DROMAJO_REQS = $(DROMAJO_H) $(DROMAJO_ROM) $(DROMAJO_DTB)
 
 firesim_lib_dir = $(firesim_base_dir)/firesim-lib/src/main/cc
 driver_dir = $(firesim_base_dir)/src/main/cc
-DRIVER_H = $(shell find $(driver_dir) -name "*.h") \
+DRIVER_H = \
+	$(shell find $(driver_dir) -name "*.h") \
 	$(shell find $(firesim_lib_dir) -name "*.h") \
 	$(DROMAJO_REQS) \
 	$(TESTCHIPIP_CSRC_DIR)/testchip_tsi.h
 
-DRIVER_CC = $(wildcard $(addprefix $(driver_dir)/, $(addsuffix .cc, firesim/*))) \
+DRIVER_CC = \
```

Comment:
This would add the platform specific implementation of simif to the driver sources. I don't think we want to do that since it would requiring having either vitis or f1 libraries installed to build metasimulators.

Reply:
Reverted.